homeassistant/components/igloohome/entity.py

```diff
@@ -0,0 +1,31 @@
+"""Implementation of a base entity that belongs to all igloohome devices."""
+
+from igloohome_api import Api
+
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class BaseEntity(Entity):
+    """A base entity that is a part of all igloohome devices."""
+
```

Comment:
```suggestion
    """A base entity that is a part of all igloohome devices."""
    
   _attr_has_entity_name = True

```
must be set for new integrations